content/guides/core/automations/create-automations/webhook.md

```diff
@@ -0,0 +1,305 @@
+
+
+{{% /tab %}}
+{{% tab "Project" %}}
+A W&B admin can view and manage a project's automations from the project's **Automations** tab.
+
+- To view an automation's details, click its name.
+- To edit an automation, click its action `...` menu, then click **Edit automation**.
+- To delete an automation, click its action `...` menu, then click **Delete automation**. Confirmation is required.
+{{% /tab %}}
+{{< /tabpane >}}
+
+## Payload reference
+Use these sections to construct your webhoook's payload. For details about testing your webhook and its payload, refer to [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}).
+
+### Payload variables
+This section describes the variables you can use to construct your webhook's payload.
+
+| Variable | Details |
+|----------|---------|
+| `${project_name}`             | The name of the project that owns the mutation that triggered the action. |
+| `${entity_name}`              | The name of the entity or team that owns the mutation that triggered the action.
+| `${event_type}`               | The type of event that triggered the action. |
+| `${event_author}`             | The user that triggered the action. |
+| `${artifact_collection_name}` | The name of the artifact collection that the artifact version is linked to. |
+| `${artifact_metadata.<KEY>}`  | The value of an arbitrary top-level metadata key from the artifact version that triggered the action. Replace `<KEY>` with the name of a top-level metadata key. Only top-level metadata keys are available in the webhook's payload. |
+| `${artifact_version}`         | The [`Wandb.Artifact`]({{< relref "/ref/python/artifact/" >}}) representation of the artifact version that triggered the action. |
+| `${artifact_version_string}` | The `string` representation of the artifact version that triggered the action. |
+| `${ACCESS_TOKEN}` | The value of the access token configured in the [webhook]({{< relref "#create-a-webhook" >}}), if an access token is configured. The access token is automatically passed in the `Authorization: Bearer` HTTP header. |
+| `${SECRET_NAME}` | If configured, the value of a secret configured in the [webhook]({{< relref "#create-a-webhook" >}}). Replace `SECRET_NAME` with the name of the secret. |
+
+### Payload examples
+This section includes examples of webhook payloads for some common use cases. The examples demonstrate how to use [payload variables]({{< relref "#payload-variables" >}}).
+
+{{< tabpane text=true >}}
+{{% tab header="GitHub repository dispatch" value="github" %}}
+
+{{% alert %}}
+Verify that your access tokens have required set of permissions to trigger your GHA workflow. For more information, [see these GitHub Docs](https://docs.github.com/en/rest/repos/repos?#create-a-repository-dispatch-event). 
+{{% /alert %}}
+
+Send a repository dispatch from W&B to trigger a GitHub action. For example, suppose you have workflow that accepts a repository dispatch as a trigger for the `on` key:
```

Comment:
Since we're talking about an external file, might be useful to be overly specific (so as to not confuse this YAML-looking code snippet with the W&B automation YAML)

```suggestion
Send a repository dispatch from W&B to trigger a GitHub action. For example, suppose you have a GitHub workflow file that accepts a repository dispatch as a trigger for the `on` key:
```